src/webserver/SessionActions.jl

```diff
@@ -30,8 +30,23 @@ function open_url(session::ServerSession, url::AbstractString; kwargs...)
     return nb
 end
 
+function convert_path_to_wsl(path)
+    "Code to check whether the system is wsl, in which case, path is changed to corresponding wsl path"
+    try
+        if Sys.islinux() && isfile("/proc/sys/kernel/osrelease") &&
```

Comment:
I've at least one fallen into an issue where this blocks if the file is open and pluto doesn't open - Investigating!

Reply:
I'm using the `detectwsl()` function instead of inlining it, even though I'm scared it blocks, because that's what microsoft suggests. I would do `uname -r` if it wasn't 'suggested'.